### ISSUE TITLE
Deprecation message while installing PINT using `pip install -e`

### ISSUE THREAD

**Issue by abhisrkckl** [nanograv/PINT#1846]
DEPRECATION: Legacy editable install of pint-pulsar==1.0.1+233.gc406b12d from file:///home/abhimanyu/Work/PINT (setup.py develop) is deprecated. pip 25.0 will enforce this behaviour change. A possible replacement is to add a pyproject.toml or enable --use-pep517, and use setuptools >= 64. If the resulting installation is not behaving as expected, try using --config-settings editable_mode=compat. Please consult the setuptools documentation for more information. Discussion can be found at https://github.com/pypa/pip/issues/11457

**Comment by abhisrkckl** [comment]
We may have to update our package setup.

**Comment by dlakaplan** [comment]
But the discussion says:

> TL;DR: the execution of setup.py develop when pip does an editable install. The -e / --editable pip install option is not deprecated.
and I (at least) do the latter.  Is this really an issue?

**Comment by dlakaplan** [comment]
https://pypi.org/project/ini2toml/ might be able to translate things for us
